android/package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "jest-appium-app-browserstack",
+    "version": "1.0.0",
+    "author": "BrowserStack",
+    "license": "MIT",
+    "description": "Appium examples using JestJS and BrowserStack",
+    "keywords": [
+      "browserstack",
+      "selenium",
+      "appium",
+      "automate",
+      "examples",
+      "jest",
+      "jestjs"
+    ],
+    "devDependencies": {
+      "@types/jest": "^27.4.0",
+      "browserstack-node-sdk": "latest",
+      "jest": "^27.4.7"
+    },
+    "dependencies": {
+      "browserstack-local": "^1.4.8",
+      "dotenv": "^16.0.0",
+      "selenium-webdriver": "4.1.2"
+    },
+    "scripts": {
+      "sample-test": "browserstack-node-sdk jest src/sample.test.js",
+      "sample-local-test": "browserstack-node-sdk jest src/sample-local.test.js",
```

```suggestion
      "sample-test": "npx browserstack-node-sdk jest src/sample.test.js",
      "sample-local-test": "npx browserstack-node-sdk jest src/sample-local.test.js",
```

To avoid picking up global node sdk if installed